### PR TITLE
Debounce

### PR DESCRIPTION
**Pull request by s0ham075**
# ✨ Codu Pull Request 💻

Codu Logo

## Pull Request details:
- debounced  article page search bar using useDebounce hook

## Any Breaking changes:
- None

## Associated Screenshots:
- None

### CODE REVIEW

File: .swc/plugins/v7_linux_x86_64_0.102.1/d560cf24a4aa929c89b44fd051c5acffb1acc62347383cb9e2fbb7eb927f8f7b
Comment (NiallJoeMaher) [inline .swc/plugins/v7_linux_x86_64_0.102.1/d560cf24a4aa929c89b44fd051c5acffb1acc62347383cb9e2fbb7eb927f8f7b:1]: Don't commit the binary, please. This is from a previous commit that I had to remove. But it should have been stopped by the `.gitignore`.

File: next.config.js [hunk 1]
```diff
@@ -50,6 +50,10 @@ const { withSentryConfig } = require("@sentry/nextjs");
 module.exports = withSentryConfig(
   module.exports,
   {
+    // For all available options, see:
```
Comment (NiallJoeMaher) [inline next.config.js:53]: Seems to be an older commit you are working off because these commits aren't needed.

File: .gitignore [hunk 2]
```diff
@@ -48,6 +48,31 @@ yarn-error.log*
 node_modules
 
 
+# CDK asset staging directory
```
Comment (NiallJoeMaher) [inline .gitignore:51]: Not sure what happened here since it looks like duplicated everything in this addition to the `gitignore`

**Review by NiallJoeMaher — CHANGES_REQUESTED** [review]
A few things to remove but looks good! 🦾

File: .swc/plugins/v7_linux_x86_64_0.102.1/d560cf24a4aa929c89b44fd051c5acffb1acc62347383cb9e2fbb7eb927f8f7b
Comment (s0ham075) [reply .swc/plugins/v7_linux_x86_64_0.102.1/d560cf24a4aa929c89b44fd051c5acffb1acc62347383cb9e2fbb7eb927f8f7b:1]: I don't understand, is this a file which needs to be removed?

Comment (NiallJoeMaher) [reply .swc/plugins/v7_linux_x86_64_0.102.1/d560cf24a4aa929c89b44fd051c5acffb1acc62347383cb9e2fbb7eb927f8f7b:1]: Yes, it's not needed. I think something is wrong with your git history so maybe you could try closing this PR and opening with a clean branch?

**Comment by NiallJoeMaher** [comment]
Closing since it seems to be stale. Feel free to reopen if it becomes active again or you have fixes 🦾 

Thanks for the help. 😄

**Pull request closed without merging**